spec/models/step_by_step_page_spec.rb

```diff
@@ -12,7 +12,6 @@
     let!(:step_by_step_page_2) { create(:step_by_step_page, slug: "a", title: "Agonise over the next title you can think of") }
 
     it 'returns step by step pages in alphabetical order by title' do
-      # binding.pry
```

Comment:
Thanks, this has been bothering me!

Reply:
Hmmm, this should have been picked up by the linter. Ideally this would be removed in a separate commit.